src/library-authoring/list-libraries/specs/LibraryListPage.spec.jsx

```diff
@@ -1,20 +1,31 @@
 import React from 'react';
+import { Router, withRouter } from 'react-router';
 import update from 'immutability-helper';
 import { BrowserRouter } from 'react-router-dom';
 import { injectIntl } from '@edx/frontend-platform/i18n';
+import { createMemoryHistory } from 'history';
 import { LibraryListPage } from '../LibraryListPage';
 import { libraryListInitialState } from '../data';
-import { LOADING_STATUS } from '../../common';
+import { LOADING_STATUS, ROUTES } from '../../common';
 import { ctxMount } from '../../common/specs/helpers';
+import { libraryFactory } from '../../common/specs/factories';
 
-const InjectedLibraryListPage = injectIntl(LibraryListPage);
+const InjectedLibraryListPage = injectIntl(withRouter(LibraryListPage));
+const history = createMemoryHistory();
 
 describe('list-libraries/LibraryListPage.jsx', () => {
   let props;
```

Comment:
It is better to define **variables** like `prop` and `historySpy` at the top of the file and reuse them in each test.